### ISSUE TITLE
Cargo.toml example

### ISSUE THREAD

**Issue by izissise** [runcc-rs/runcc#3]
Hello, I couldn't make this work by writing the configuration in `Cargo.toml`.
I keep getting this error 
```
Config file error: Invalid config file Cargo.toml: no field
```
Looking at the rust source it is not obvious what field I'm missing, could you provide and example where the runcc config is in Cargo.toml? 

Thanks,
Hugues

**Comment by EqualMa** [comment]
Hi, this should be fixed by `runcc@v2.0.3`.
Please run `cargo install runcc` to update it.

Here is an example of `Cargo.toml`:

```toml
[package.metadata.runcc.commands]
success1 = 'node -e "setTimeout(()=>{},700)"'
success2 = 'node -e "setTimeout(()=>{},300)"'
success3 = "node -e \"''\""
```

Change `package` to `workspace` is this Cargo.toml is a workspace root.

**Comment by izissise** [comment]
Thanks a lot!
